### PR TITLE
Make commands look more like notion

### PR DESCRIPTION
**Pull request by valentinludu**
When you type / to see all the commands inside a page you will see the list of commands look like the Notion ones.

I discussed with Chris and he approved the design and how the icons look.

### CODE REVIEW

File: components/common/CharmEditor/components/inlinePalette/editorItems/text.tsx [hunk 1]
```diff
@@ -178,7 +180,7 @@ export function items (): PaletteItemTypeNoGroup[] {
...
       return {
         uid: `headingConvert${level}`,
-        icon: <svg stroke='currentColor' fill='currentColor' strokeWidth={0} viewBox='0 0 512 512' height='1em' width='1em' xmlns='http://www.w3.org/2000/svg'><path d='M448 96v320h32a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16H320a16 16 0 0 1-16-16v-32a16 16 0 0 1 16-16h32V288H160v128h32a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16H32a16 16 0 0 1-16-16v-32a16 16 0 0 1 16-16h32V96H32a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16h-32v128h192V96h-32a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16z' /></svg>,
+        icon: <svg stroke='currentColor' fill='currentColor' strokeWidth={0} viewBox='0 0 512 512' height='30px' width='30px' xmlns='http://www.w3.org/2000/svg'><path d='M448 96v320h32a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16H320a16 16 0 0 1-16-16v-32a16 16 0 0 1 16-16h32V288H160v128h32a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16H32a16 16 0 0 1-16-16v-32a16 16 0 0 1 16-16h32V96H32a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16h-32v128h192V96h-32a16 16 0 0 1-16-16V48a16 16 0 0 1 16-16h160a16 16 0 0 1 16 16v32a16 16 0 0 1-16 16z' /></svg>,
```
Comment (msadura) [inline components/common/CharmEditor/components/inlinePalette/editorItems/text.tsx:183]: can't you reuse IconSize here as well?

File: components/common/CharmEditor/components/inlinePalette/components/InlinePaletteRow.tsx [hunk 2]
```diff
@@ -79,8 +82,17 @@ export default function InlinePaletteRow ({
       className={className}
       style={style}
     >
-      <Box display='flex' sx={{ color: 'secondary.light' }} component='span'>{icon}</Box>
-      {title}
+      <Paper
+        elevation={0}
+        variant='outlined'
+        sx={{ width: 46, height: 46, display: 'flex', alignItems: 'center', justifyContent: 'center', color: 'secondary.light', marginLeft: '10px' }}
```
Comment (msadura) [inline components/common/CharmEditor/components/inlinePalette/components/InlinePaletteRow.tsx:88]: I think component with this many inline styles  should be converted to styled component

File: components/common/CharmEditor/components/PopoverMenu.tsx [hunk 3]
```diff
@@ -14,7 +14,9 @@ interface Props {
 
 const StyledPaper = styled(Paper)<{ maxheight: string }>`
   max-height: ${(props) => props.maxheight};
+  width: 320px;
```
Comment (msadura) [inline components/common/CharmEditor/components/PopoverMenu.tsx:17]: Is it safe? I mean won't it break or look weird on mobile?

Comment (valentinludu) [reply components/common/CharmEditor/components/PopoverMenu.tsx:17]: Tested on mobile and looks good, even on a small iPhone SE with 375px width screen.

File: components/common/CharmEditor/components/inlinePalette/components/InlinePaletteRow.tsx (hunk 2 above)
Comment (valentinludu) [reply components/common/CharmEditor/components/inlinePalette/components/InlinePaletteRow.tsx:88]: Yep, good idea.

File: components/common/CharmEditor/components/inlinePalette/editorItems/text.tsx (hunk 1 above)
Comment (valentinludu) [reply components/common/CharmEditor/components/inlinePalette/editorItems/text.tsx]: Yes, I missed that.

**Review by msadura — APPROVED** [review]

**Pull request merged**